src/jwt-server.js

```diff
@@ -76,12 +76,13 @@ const p: FusionPlugin<SessionDeps, SessionService> =
       secret: SessionSecretToken,
       cookieName: SessionCookieNameToken,
       expires: SessionCookieExpiresToken.optional,
+      getFullPath: GetFullPathToken.optional,
     },
     provides: deps => {
-      const {secret, cookieName, expires = 86400} = deps;
+      const {secret, cookieName, expires = 86400, getFullPath = (ctx: Context) => keyPath => `data.${keyPath}`} = deps;
       const service: SessionService = {
         from: memoize((ctx: Context) => {
-          return new JWTSession(ctx, {secret, cookieName, expires});
+          return new JWTSession(ctx, {secret, cookieName, expires, getFullPath(ctx)});
```

Comment:
Currently this is specified as part of `config` here, but specified as another constructor parameter in `JWTSession`. I'm assuming that since `config` is fairly session specific, that this should be another parameter here instead of in config.

Reply:
@KevinGrandon Agree, so in this case we will sepaprate `geFullPath` out of config object. Thanks.